ext/xsl/php_xsl.stub.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+class XSLTProcessor
+{
+    /** @return bool */
+    public function importStylesheet(object $stylesheet) {}
+
+    /** @return DOMDocument|false */
+    public function transformToDoc(DOMNode $document, ?string $return_class = null) {}
+
+    /**
+     * @param DOMDocument|SimpleXMLElement $document
+     * @return int
+     */
+    public function transformToUri($document, string $uri) {}
+
+    /**
+     * @param DOMDocument|SimpleXMLElement $document
+     * @return string|false|null
+     */
+    public function transformToXml(object $document) {}
+
+    /**
+     * @param string|array $name
+     * @return bool|null
```

Comment:
I don't think null is possible here.

Reply:
that must have been an oversight on my part